### DISCUSSION TITLE
How to use gui.update_idletasks() in CustomTkinter

### DISCUSSION THREAD

**Discussion by Barry-sze** [TomSchimansky/CustomTkinter#414] [General]
def StartProgress():
    for i in range(5):
        progress_var['value'] += 20
        stringvar.set(progress_var['value'])
        # get the value
        label.config(text=stringvar.get()+' %')
        # update value
        gui.update_idletasks()
        time.sleep(1)

**Comment by TomSchimansky** [comment]
Do you want to move the progress bar slowly over time? Why do you want to use `i.update_idletasks()`? I think I already answered to this kind of problem here: https://github.com/TomSchimansky/CustomTkinter/discussions/394#discussioncomment-3413055
